sys/include/net/gnrc/tcp/config.h

```diff
@@ -26,6 +28,28 @@
 extern "C" {
 #endif
 
+/**
+ * @defgroup net_gnrc_tcp_conf GNRC TCP compile configurations
+ * @ingroup net_gnrc_conf
+ *
+ * ## Calculating RTO
+ * To calculate retransmission timeout (RTO), Round Trip Time (RTT) needs to be
+ * taken into account. SRTT (smoothed round-trip time) and RTTVAR (round-trip
+ * time variation) are hence calculated as follows:
+ *
+ *     RTTVAR <- (1 - beta) * RTTVAR + beta * |SRTT - R'|
+ *     SRTT <- (1 - alpha) * SRTT + alpha * R'
+ *
+ * where alpha and beta are constants, and R' is the instantaneous RTT value.
+ *
+ * RTO is then calculated as :
+ *
+ *     RTO <- SRTT + max (G, K*RTTVAR)
+ *
+ * where K is a constant, and G is clock granularity in seconds.
```

Comment:
Maybe you can add references to the macros that represent these parameters

Reply:
Done.